utPlayer.py

```diff
@@ -90,7 +116,7 @@ def stop(self):
     # test the class
     shell = InteractiveShell()
 
-    test = utPlayer(shell)
+    test = utPlayer(shell, "element", "gstreamer")
```

Comment:
`def __init__(self, session:object, vendor:str, log:logModule=None):`

don't think this is correct anymore -> `utPlayer(shell, "element", None );`

Reply:
update the test code